lib/gis/gisinit.c

```diff
@@ -49,12 +49,14 @@ void G__gisinit(const char *version, const char *pgm)
 
     G_set_program_name(pgm);
 
+#ifndef _WIN32
```

Comment:
is this intended to work on all other os, but not on windows? why?

Reply:
This is triggered whenever something (eg. the QGIS GRASS plugin) is built against a slightly different version of GRASS (ie. a different commit).  But the DLLs carry the major and minor version number in their name on Windows, so this could only be a problem if the API/ABI is broken in a point release - which IMHO is unlikely.

Reply:
API/ABI should change in point version. @ninsbl I suggest to remove this part from PR.

Reply:
> Yes, API/ABI shouldn't be change in point version. @ninsbl I suggest to remove this part from PR.

huh?  why then remove the part?